src/repository/employee.go

```
@@ -0,0 +1,45 @@
+
+	result := u.db.Model(&employee).Where("id=?", id).Scan(&employee)
+	if result.Error != nil && errors.Is(result.Error, gorm.ErrRecordNotFound) {
+		return nil, genericerror.NewGenericError(constant.ErrorCodeResourceNotFound, result.Error.Error(), nil, result.Error)
+	}
+
+	if result.Error != nil {
+		return nil, genericerror.NewInternalErrByErr(result.Error)
+	}
+
+	if result.RowsAffected == 0 {
+		return nil, genericerror.NewGenericError(constant.ErrorCodeResourceNotFound, "No employee found", nil, errors.New("No employee found"))
```

golangci-lint (1.64.8)

The line is 137 characters long, which exceeds the maximum of 120 characters.

(lll)